modules/era/sql/era_droplist.sql

```diff
@@ -13381,8 +13341,8 @@ INSERT INTO `mob_droplist` VALUES (1672,0,0,1000,1456,@VRARE);   -- One Hundred
 -- ZoneID: 160 - Million Eyes
 INSERT INTO `mob_droplist` VALUES (1673,0,0,1000,939,@VCOMMON); -- Hecteyes Eye (Very Common, 24%)
 INSERT INTO `mob_droplist` VALUES (1673,0,0,1000,1288,@RARE);   -- Wooden Hakutaku Eye (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1673,0,0,1000,4754,@RARE);   -- Scroll Of Fire Iii (Rare, 5%)
-INSERT INTO `mob_droplist` VALUES (1673,0,0,1000,4784,@RARE);   -- Scroll Of Firaga Iii (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (1673,0,0,1000,4754,@VRARE);  -- Scroll Of Fire Iii (Very Rare, 1%)
+INSERT INTO `mob_droplist` VALUES (1673,0,0,1000,4784,@VRARE);  -- Scroll Of Firaga Iii (Very Rare, 1%)
```

Comment:
Same as above.
http://ffxidb.com/zones/160/million-eyes
If not I would make Fire III 5% at least. 1% seems extremely unlikely for both of these